src/components/pages/footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import './Footer.css'
+
+function Footer () {
+    return (
+        <footer className="footer"> 
+        <hr className="bar"/>
+        Made with Love for FOSS in India
+        </footer>
```

Comment:
Can we use tailwind class names instead of using custom CSS? Feel free to let me know if you would like to discuss this further on how to go about it in case you are new to tailwind css.